lib/solana.cpp

```diff
@@ -70,12 +79,42 @@ Keypair Keypair::fromFile(const std::string &path) {
   return result;
 }
 
+uint64_t trailingZeros(uint64_t n) {
+  int trailingsZeros = 0;
```

Comment:
type should be same as return type.

Reply:
Changed variable trailingsZeros type to` uint64_t`